src/app/student-display/student-display.component.html

```diff
@@ -0,0 +1,36 @@
+<ng-container [ngSwitch]="currentScreen$ | async">
+  <ng-container *ngSwitchCase="ScreenId.SessionLobby">
+    <app-hill-background cover></app-hill-background>
+    <div class="centered-container">
+      <mat-card>
+        <mat-card-title>Waiting for round to begin&hellip;</mat-card-title>
+      </mat-card>
+    </div>
+  </ng-container>
+
+  <ng-container *ngSwitchCase="ScreenId.PlayRound">
+    <app-play-round></app-play-round>
+  </ng-container>
+
+  <ng-container *ngSwitchCase="ScreenId.NoSession">
+    <app-hill-background cover></app-hill-background>
+    <div class="centered-container">
+      <mat-card>
+        <mat-card-title>Error: invalid session</mat-card-title>
```

Comment:
We were thinking about that and decided that we want to consider the design of an "exit" button for the entire student display, including AR, and that this could be added later.